packages/rancher-monitoring/generated-changes/patch/values.yaml.patch

```diff
@@ -526,15 +526,25 @@
    ## Deploy default dashboards.
    ##
    defaultDashboardsEnabled: true
```

Comment:
I considered modifying `defaultDashboardsEnabled` to `defaultDashboards.enabled` but I didn't want to introduce a breaking change if a user had previously set `defaultDashboardsEnabled=false`.